### PR TITLE
Target .NET Core 2

### PR DESCRIPTION
**Pull request by NextTurn**
Why not?

### CODE REVIEW

**Comment by KexyBiscuit** [comment]
Because .NET Core 2.0 is [out of support](https://github.com/dotnet/core/blob/master/microsoft-support.md#out-of-support-net-core-releases).

**Comment by NextTurn** [comment]
Thanks @KexyBiscuit. Target .NET Core 2.2 then.

File: samples/dotnetsay/dotnetsay.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (karelz) [inline samples/dotnetsay/dotnetsay.csproj]: The sample README says 2.1 or higher (twice) - maybe it is ok to leave it at 2.1? Shouldn't it build just fine even if only 2.2 is installed?

Comment (NextTurn) [reply samples/dotnetsay/dotnetsay.csproj]: I guess 2.0 is just OK.

**Pull request merged**